library/Icinga/Application/ProvidedHook/DbMigration.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+/* Icinga Web 2 | (c) 2023 Icinga GmbH | GPLv2+ */
+
+namespace Icinga\Application\ProvidedHook;
+
+use Icinga\Application\Hook\DbMigrationHook;
+use Icinga\Common\Database;
+use Icinga\Model\Schema;
+use ipl\Orm\Query;
+use ipl\Sql\Connection;
+
+class DbMigration extends DbMigrationHook
+{
+    use Database {
+        getDb as public getPublicDb;
```

Comment:
My initial goal was to just change the visibility. (Without needing to override the method)

Now the visibility of public doesn't make sense anymore, also the name of the alias does not.

Use `as private getWebDb;` instead. The alias shouldn't be public. It's for internal use after all.